<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tabla_cuentas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>271</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string># de cuenta</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Estado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Balance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Id Propietario</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
